aea/protocols/default/custom_types.py

```diff
@@ -35,12 +35,29 @@ class ErrorCode(Enum):
     def encode(
         cls, performative, error_code_from_message: "ErrorCode",
     ):
-        """Encode an instance of this class into its protobuf object."""
+        """
+        Encode an instance of this class into the protocol buffer object.
+
+        'performative' argument is the protocol buffer object containing a content whose type is this class.
```

Comment:
This is a repetition of the parameter definition, I'd remove line 41

Reply:
Done